src/containers/Header/Search.jsx

```diff
@@ -77,9 +77,8 @@ class Search extends Component {
     this.onKeyDown = this.onKeyDown.bind(this);
   }
 
-  /* eslint-disable */
+  // eslint-disable-next-line
```

Comment:
```suggestion
  // eslint-disable-next-line camelcase
```
This disables the next line just for that eslint rule, instead of all of them. If you look at the fix options for this error in VS Code, it'll suggest that as the first one. (if you don't know how to do that I can show you)